integration-tests/utils/integration-helpers.js

```diff
@@ -36,7 +36,8 @@ const makeWorkspace = withSignedInPage(async ({ page, billingProject }) => {
 
     console.info(`Created workspace: ${workspaceName}`)
   } catch (e) {
-    throw Error(`Failed to create workspace: ${workspaceName} with billing project ${billingProject}`)
+    console.error(`Failed to create workspace: ${workspaceName} with billing project: ${billingProject}`, e)
```

Comment:
Unfortunately I don't think you get anything useful from the error itself either… I looked into this previously. You can search in kibana though to see the cause of the issue.

Reply:
Log search in Kibana is a nightmare. Do you know why `window.Ajax().Workspaces.create` does not throw meaningful stacktrace?

Reply:
No, I don't know why. I found searching in Kibana not so difficult because we have the workspace ID. That's how I discovered the problem of workspace names getting reused (when the workspace hadn't been properly deleted in the past).

Reply:
I think it might be worth merging this after all, as I now suspect that there may be errors that aren't actually related to the server delete call.

Reply:
Screenshot is a white blank page in failed CircleCI [job](https://app.circleci.com/pipelines/github/DataBiosphere/terra-ui/9579/workflows/9b873798-8238-4730-9712-76331d3e57ad) this morning. I have a theory. It may be related to `window.Ajax().Workspaces.create()` because traffics shown in logs seem to indicate page has loaded okay.